docs/commands/lmmreg.md

```diff
@@ -0,0 +1,173 @@
+    -c 'variant = v, va.lmmreg.*' \
+    -o lmmreg.tsv.bgz
+```
+
+Similarly `lmmreg` does *not* remove samples from the underlying variant dataset.
+
+#### Performance
+
+This initial version of `lmmreg` scales efficiently to tens of thousands samples and an unbounded number of variants, making it particularly well-suited to modern sequencing studies. For example, starting from a VDS of the 1000 Genomes Project (consisting of 2535 whole genomes), `lmmreg` computes a kinship matrix based on 100k common variants, fits coefficients and variance components in the sample-covariates-only model, runs a linear-mixed-model likelihood ratio test for all 15 million high-quality non-rare variants, and exports the results in under 4 minutes. Here we used 42 preemptible workers (~680 cores) on 2k partitions at a cost of about 50 cents on Google cloud (see [Using Hail on the Google Cloud Platform](http://discuss.hail.is/t/using-hail-on-the-google-cloud-platform/80)). The first analysts to apply `lmmreg` in research have found the method takes under an hour on 8185 samples and 25 million variants, with 262k variants used to compute kinship.
+
+While `lmmreg` computes the kinship matrix $K$ using distributed matrix multiplication (Step 2), the full eigendecomposition (Step 3) is currently run on a single core of master using the [LAPACK routine DSYEVD](http://www.netlib.org/lapack/explore-html/d2/d8a/group__double_s_yeigen_ga694ddc6e5527b6223748e3462013d867.html), which we empirically find to be the most performant of the four available routines; laptop performance plots showing cubic complexity in $n$ are available [here](https://github.com/hail-is/hail/pull/906). On Google cloud, eigendecomposition takes about 2 seconds for 2535 sampes and 1 minute for 8185 samples. If you see worse performance, check that LAPACK natives are being properly loaded (see "BLAS and LAPACK" in [Getting Started](https://hail.is/getting_started.html)). We will soon add functionality to save intermediate results so they need not be recalculated when following up on genomic regions of interest.
+
+Given the eigendecomposition, fitting the global model (Step 4) takes on the order of a few seconds on master. Association testing (Step 5) is fully distributed by variant with per-variant time complexity that is completely independent of the number of sample covariates and dominated by multiplication of the genotype vector $v$ by the matrix of eigenvectors $U^T$ as described below, which we accelerate with a sparse representation of $v$.  The matrix $U^T$ has size about $8n^2$ bytes and is currently broadcast to each Spark executor. For example, with 15k samples, storing $U^T$ consumes about 3.6GB of memory on a 16-core worker node with two 8-core executors . So for large $n$, we recommend using a high-memory configuration such as `highmem` workers.
+
+Future versions of `lmmreg` will scale to far more samples with exact inference and offer acceleration through approximate inference.
```

Comment:
Since the paragraph starts with "this initial version of lmmreg", I feel fine taking this out:

"We plan future versions of `lmmreg` will scale to far more samples with exact inference and offer acceleration through approximate inference."